src/pages/docs/step-cli/the-step-command.mdx

```diff
@@ -41,6 +41,38 @@ The file `$STEPPATH/config/defaults.json` can contain a JSON object listing defa
     It is not currently possible to segment default keys in <Code>defaults.json</Code> by subcommand. All keys are applied to all <Code>step</Code> commands.
 </Alert>
 
+## Working with multiple CAs
+
+By default, the `step` toolchain assumes a single CA is in use, and it will store that CA's configuration and state directories (`config`, `certs`, `secrets`, `db`, etc.) inside a single folder, `$HOME/.step`.
+
+If you regularly work with multiple CAs, we recommend enabling _contexts_.
+
+Contexts let you configure and select multiple CAs and configuration profiles. If this optional feature is enabled, the `$HOME/.step` folder will have a different layout.
+
+Every context consists of a configuration profile directory (which includes user-managed [`defaults.json`](#configuration-file) and `ca.json`), and a directory for CA-managed data (CA roots, certificates, secrets, templates).
```

Comment:
There is not user managed ca.json. Only the defaults.json. There's only one `ca.json` and it's under the authorities. I could see why someone might think that's "user-managed" though.